### ISSUE TITLE
Please provide Dockerfiles for all used container

### ISSUE THREAD

**Issue by webwurst** [polvi/fleet-redis-demo#4]
I would like to work through all the details for a better understanding.
Maybe you could make use of the [Trusted Builds](http://blog.docker.io/2013/11/introducing-trusted-builds/) feature?

**Comment by litch** [comment]
:thumbsup: I second this please.  Very cool system, but I'd love to see how it's built.

**Comment by blakelapierre** [comment]
Yes, please provide the Dockerfiles. This demo is used on the coreos.com page, but the example isn't complete without the Dockerfiles.
